Samantha-samanthalpark/Homework-3/object-homework/app.js

```diff
@@ -0,0 +1,94 @@
+/////////////////////////////////////////////////////////////
+
+///////////////////////// Challenge /////////////////////////
+
+/////////////////////////////////////////////////////////////
+
+// Write your answers inside this file, at the places where it's indicated by the comments.
+
+
+// 1. Suppose that we wanted to create a browser-based calendar program for keeping track of things. In comments, list at least three reasonable abstractions that you might use to build this program; for each abstraction, list out at least two properties and behaviors that it would make sense for that abstraction to have. Below is an example - please write your answer in the same format.
+  // 'Car'
+  // Description: Our app manages car sales.
+  // Every 'car' has
+  // - a make
+  // - a model
+  // - a year
+  // Every 'car' can
+  // - drive
+  // - brake
+  // - park
+
+// Answer Starts Here
+
+// 'Calendar'
```

Comment:
I like how detailed your abstraction is here. This is really helpful when building out code. Nicely done!